tools/depends/target/libicundk/Makefile

```diff
@@ -0,0 +1,39 @@
+include ../../Makefile.include
+DEPS=../../Makefile.include Makefile ../../download-files.include
+
+# lib name, version
+LIBNAME=libicundk
```

Comment:
how future-proof is relying on libicundk? are there any alternatives now? I can see that the last commit was 3 years ago?

Reply:
I found that ICU4C has been available in recent NDKs. Will check if it's available for our use when I have time. See https://developer.android.com/guide/topics/resources/internationalization#icu4c

Reply:
Looks like ICU4C is only available on Android 12 and NDK r22b, and not available in current toolchains of kodi.

Reply:
If we package a self-contained ICU and only include the collation data, looks like it would only take a few MBs. Is it acceptable?